### DISCUSSION TITLE
AVG flags with Trojan

### DISCUSSION THREAD

**Discussion by ghost** [jziolkowski/tdm#283] [General]
FYI, this recent release was flagged by AVG Free as having a trojan. VirusTotal gave it an all-clear, so the problem may be with AVG (I flagged it in AVG as a false positive). This is FYI in case others come across the same issue.

**Comment by jziolkowski** [comment]
Known problem with windows-packaged python applications.

https://discuss.python.org/t/pyinstaller-false-positive/43171/15
